scripts/format_app.py

```diff
@@ -0,0 +1,70 @@
+# format_app.py
+
+import argparse
+import os
+import subprocess
+import sys
+
+
+def handle_cli_args():
+  parser = argparse.ArgumentParser(description='A script for formatting a given app within emqx. Example Usage: /bin/python3.9 scripts/format_app.py -a emqx_resource -b format_resource_app')
```

Comment:
Nit: `argparse` already prints the usage example when called with `-h` or with something wrong.